src/main/java/seedu/stockpal/data/product/Product.java

```diff
@@ -53,4 +69,9 @@ public void decreaseQuantity(Integer amountToChange) {
             Ui.printToScreen("Insufficient amount in inventory. Amount in inventory: " + quantity);
         }
     }
+
+    @Override
+    public String toString() {
+        return "" + getPid() + getName() + getQuantity() + getPrice() + getDescription();

```

Comment:
There's no need to use getters if you are calling the variables within the class itself